src/libYARP_dev/CMakeLists.txt

```diff
@@ -84,7 +84,6 @@ set(YARP_dev_HDRS include/yarp/dev/all.h
                   include/yarp/dev/ServerSerial.h
                   include/yarp/dev/ServerSoundGrabber.h
                   include/yarp/dev/ServiceInterfaces.h
-                  include/yarp/dev/TestFrameGrabber.h
```

Comment:
One small issue...
`TestFrameGrabber.h` is a public API in master. Should we deprecate it instead of removing it?
@randaz81 @pattacini @barbalberto @traversaro